src/ServiceControl.Persistence.RavenDB/RavenBootstrapper.cs

```diff
@@ -10,6 +10,6 @@ static class RavenBootstrapper
         public const string MinimumStorageLeftRequiredForIngestionKey = "MinimumStorageLeftRequiredForIngestion";
         public const string DatabaseNameKey = "RavenDB5/DatabaseName";
         public const string LogsPathKey = "LogPath";
-        public const string LogsModeKey = "LogMode";
+        public const string RavenDbLogLevelKey = "RavenDBLogLevel";
```

Comment:
Is the name of the setting the same for Audit and Error instances?

Reply:
Yes, see Particular/ServiceControl/src/ServiceControl.Audit.Persistence.RavenDB/RavenPersistenceConfiguration.cs